src/scenic/core/vectors.py

```diff
@@ -324,6 +324,12 @@ def _inverseRotation(self):
 
     # will be converted to a distributionMethod after the class definition
     def __mul__(self, other) -> Orientation:
+        """To compose rotations, i.e. apply rotation A followed by rotation B,
```

Comment:
This doesn't have the right format for a docstring (a la [PEP 257](https://peps.python.org/pep-0257/), which the documentation builder expects): the current discussion should be in a second paragraph, with a 1-line summary at the top separated by a blank line. Like:
```
"""Apply a rotation to this orientation, yielding a new orientation.

As we represent orientations as intrinsic rotations, rotation A followed by rotation B is
given by the quaternion product A*B, not B*A. See...
"""
```